### ISSUE TITLE
<my domain>/apps/notes/notes - 404 Not Found

### ISSUE THREAD

**Issue by stevleibelt** [owncloud/notes#234]
Hi there,

after an update to nextcloud and an included update to the app "notes" to version 2.1.0, all I get is an blank screen.
A check on my developer bar shows the issue from the subject:

```
Not Found

The requested URL /apps/notes/notes was not found on this server.
```

Anything I can do?
I did a grep for "notes/notes" on the filesystem but could not find anything fitting.

**Comment by marcin-gryszkalis** [comment]
updating to git version soved that for me (see https://github.com/owncloud/notes/pull/217 )

**Comment by bdillahu** [comment]
Latest git (as of today) didn't work for me... 

I wound up putting a link to itself in the notes directory:

cd ..../apps/notes
ln -s . notes

Which makes it work.

**Comment by Henni** [comment]
I can't reproduce this. Please give me a bit more information on your configurations.

**Comment by stevleibelt** [comment]
In between, I switched to nextcloud 9.0.53, without any changes.

I am selecting "Notes" in the "App"'s-Navigation, and the url "https://<my doman>/index.php/apps/notes/#/"" is opened.
The rest is described above.

What kind of information do you need?

Thanks in advance

**Comment by Henni** [comment]
@stevleibelt which version of the app are you using?
I released 2.0.2 today, which might fix this.
